private/buf/bufgen/generator.go

```diff
@@ -562,6 +562,7 @@ func newModifier(
 		sweeper,
 		javaMultipleFilesValue,
 		managedConfig.Override[bufimagemodify.JavaMultipleFilesID],
+		false, // preserveExistingValue
```

Comment:
Not exposing this to `buf generate` ATM - planning to use in the BSR for remote packages.